includes/commerce_pos.common.inc

```diff
@@ -53,6 +53,9 @@ function commerce_pos_product_search_api_search($keywords) {
     if ($index_id != 'default') {
       $index = search_api_index_load($index_id);
       $query = new SearchApiQuery($index);
+      if (variable_get('commerce_pos_search_published_only') && isset($index->getFields()['status'])) {
```

Comment:
somewhat pedantic, but should probably default to FALSE for both these variable_get calls

Reply:
oh ya good catch, I'll change that up.